notebooks/sippy-analysis.ipynb

```diff
@@ -0,0 +1,2411 @@
+{
```

Comment:
<p>Is this the best way to get the data? Why aren't we relying on our own data gathering tools from <code>notebooks/data-sources/TestGrid/testgrid_EDA.ipynb</code>  or using our test data at <code>data/raw/</code>?  </p><p> </p><p>Seems to rely too much on Sippy work </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>